src/gwkokab/inference/jenks.py

```
@@ -0,0 +1,318 @@
+# Copyright 2023 The GWKokab Authors
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)